alvr/dashboard/js/app/main.js

```diff
@@ -37,6 +37,11 @@ define([
         const template = compiledTemplate(i18n);
 
         function checkForUpdate(settings, delay) {
+            // Linux uses package managers or manual updates
+            if (/Linux/.test(window.navigator.platform)) {
```

Comment:
This is more readable:
```suggestion
            if (window.navigator.platform.includes("Linux")) {
```

Reply:
Oh and I'm not sure if this commit should be kept in: it breaks the case of a remote Windows ALVR being accessed over the network from Linux.